templates/zerver/integrations/index.html

```diff
@@ -140,7 +140,7 @@ <h4 class="integration-category">{{ integration.categories[0] }}</h4>
                     </div>
                 </div>
 
-                <div id="integration-instructions-group">
+                <div id="integration-instructions-group" class="markdown">
```

Comment:
Shouldn't this be on the `integration-instructions` element? I don't think the categories section should be inside this class either.

Reply:
Yeah, fixed!